saleor/checkout/complete_checkout.py

```diff
@@ -164,7 +165,10 @@ def _release_checkout_voucher_usage(
     )
 
     checkout.is_voucher_usage_increased = False
-    checkout.save(update_fields=["is_voucher_usage_increased"])
+    if update_fields is None:
```

Comment:
Maybe worth to add `checkout` as a prefix? The function is related to release voucher usage, and at the beginning it wasn't clear to me that we're operating on checkout's update_fields